scriptworker/constants.py

```diff
@@ -149,13 +149,19 @@
 
     # decision task cot
     "valid_decision_worker_types": (
-        "gecko-decision",
+        "gecko-1-decision",
+        "gecko-2-decision",
+        "gecko-3-decision",
+        "gecko-decision", # legacy
```

Comment:
Linting requires 2 spaces before an inline comment. Also, is there a timeframe or bug to follow for when `gecko-decision` can be removed?